scripts/sct_deepseg.py

```diff
@@ -46,6 +46,21 @@ def get_parser():
              "suffix '_seg' will be added and output extension will be .nii.gz.",
         metavar=sct.utils.Metavar.str)
 
+    seg = parser.add_argument_group('\nTASKS')
+    seg.add_argument(
+        "-task",
+        help="Task to perform. It could either be an official SCT task (in that case, simply enter the name of the "
+             "task, example: -task segment_t2star_sc). To list official tasks, run: sct_deepseg -list-tasks.",
```

Comment:
sorry i didn't deal with PEP8 carriage return in this comment
```suggestion
        help="Task to perform. It could either be a pre-installed task, task that could be installed, or a custom task. To list available tasks, run: sct_deepseg -list-tasks",
```